app/assets/stylesheets/administrate/application.scss

```diff
@@ -10,3 +10,5 @@
 @import "library/**/*";
 @import "base/**/*";
 @import "components/**/*";
+
+@import "administrate/customize";
```

Comment:
Is this import working? I believe `customize.scss` is a sibling to this manifest file and so the import should be:

```scss
@import "customize";
```

Also, what about naming it `customizations`?